src/main/java/com/aws/greengrass/clientdevices/auth/certificate/CertificateStore.java

```diff
@@ -265,29 +263,6 @@ private void setCaKeyAndCertificateChain(Key privateKey, Certificate... caCertif
         setCaKeyAndCertificateChain(CertificateHelper.ProviderType.DEFAULT, privateKey, certificates);
     }
 
-
```

Comment:
These methods have more or less just been moved (and re-worked ever so slightly)